gazebo/rendering/DepthCamera.hh

```diff
@@ -92,6 +92,14 @@ namespace gazebo
       /// \return The z-buffer as a float array
       public: virtual const float *DepthData() const;
 
+      /// \brief Get whether reflectance has been generated.
+      /// \return True if reflectance is set to generate.
+      public: bool GetOutputReflectance() const;
+
+      /// \brief All things needed to get reflectance data
+      /// \return The reflectance buffer as a float array
```

Comment:
can you clarify what the values in this array mean physically? does 0 mean nonreflective and 1 mean reflective?

I'm asking because you inverted the data in c776173ca7ff3b115bb691bb40499e2ed1d78621

Reply:
oh we should not invert the data on the sensors side. We should only update the visualization code. The gui visualization inverts the data when it detects that the underlying image has a FLOAT type:
https://github.com/osrf/gazebo/blob/gazebo11/gazebo/gui/viewers/ImageFrame.cc#L157-L167

The logic there was added for visualization of depth data, i.e. black pixels means far away while white means close to camera. However, now that the reflectance image output is also in FLOAT, we don't want the inversion to be applied to the reflectance image. I think there needs to be some way on the gui side to tell reflectance images from depth images so that the correct visualization is applied.